File: src/main/java/io/github/dqualizer/dqlang/types/apiadapter/ServerInfo.java
package io.github.dqualizer.dqlang.types.apiadapter;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class ServerInfo {

    private String host;
    private String environment;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/openapiadapter/dqlang/ServerInfo.kt
package io.github.dqualizer.dqlang.archive.openapiadapter.dqlang

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class ServerInfo {
    val host: String? = null
    val environment: String? = null
}
